swarm-derive/src/lib.rs

```diff
@@ -896,7 +899,20 @@ fn parse_attributes(ast: &DeriveInput) -> Result<BehaviourAttributes, TokenStrea
                 continue;
             }
 
-            if meta.path().is_ident("out_event") {
+            if meta.path().is_ident("to_swarm") || meta.path().is_ident("out_event") {
+                if meta.path().is_ident("out_event") {
+                    is_out_event = true;
+
+                    let meta_token_stream = meta.to_token_stream();
+                    let input = syn::parse2::<syn::DeriveInput>(meta_token_stream.into()).unwrap();
+                    let warning = proc_macro_warning::Warning::new_deprecated("test")
```

Comment:
@thomaseizinger I tried the implementation here, still not working, can you have a look? Maybe you will find a solution, I think we are quite close but I miss a piece of the puzzle... :)